src/paper-eval/linear-algebra.dsl

```diff
@@ -0,0 +1,16 @@
+tconstructor Scalar : type
+tconstructor VectorSpace : type
+tconstructor Vector : type
+tconstructor LinearMap : type
+
+operator Add (v1 : Vector, v2 : Vector) : Vector
```

Comment:
now we want functions to have lowercase names (it's OK for predicates to have uppercase names, i think)